datacats/environment.py

```diff
@@ -489,11 +489,12 @@ def _generate_passwords(self):
             'DATASTORE_RW_PASSWORD': generate_db_password(),
             }
 
-    def start_web(self, production=False):
+    def start_web(self, host='127.0.0.1', production=False):
```

Comment:
call this address and add it to the end of the args, just in case some other code is depending on production being the first arg.